client/state/ui/editor/media/actions.js

```diff
@@ -1,11 +1,28 @@
 /**
  * Internal dependencies
  */
-import { EDITOR_MEDIA_EDIT_ITEM_SET } from 'state/action-types';
+import {
+	EDITOR_MEDIA_EDIT_ITEM_SET,
+	IMAGE_EDITOR_STATE_CHANGE,
+	IMAGE_EDITOR_STATE_RESET
+} from 'state/action-types';
 
 export function setEditorMediaEditItem( item ) {
 	return {
 		type: EDITOR_MEDIA_EDIT_ITEM_SET,
 		item
 	};
 }
+
+export function setImageEditorState( newState ) {
```

Comment:
This would be more useful if there were more fine grained. For example a flip action, a rotate action, a crop action.